mmdeploy/apis/vacc/__init__.py

```diff
@@ -0,0 +1,14 @@
+# Copyright (c) OpenMMLab. All rights reserved.
+from mmdeploy.backend.vacc import from_onnx as _from_onnx
+from mmdeploy.backend.vacc import is_available
+from ..core import PIPELINE_MANAGER
+
+from_onnx = PIPELINE_MANAGER.register_pipeline()(_from_onnx)
+
+__all__ = ['is_available', 'from_onnx']
+
+if is_available():
+    try:
```

Comment:
I'm afraid `from_onnx` should be registered here.

Reply:
`from_onnx` defined at `onnx2vacc`，why should registered here again？

Reply:
`is_available` is used to check if backends available. `from_onnx` is accessible only if the backend available. [This](https://github.com/open-mmlab/mmdeploy/blob/cbddf5a458e75ef47f7b9d1ef7cb74d39b7d81f7/mmdeploy/apis/tensorrt/__init__.py#L10) for example.